packages/picongpu/picongpu.profile

```diff
@@ -7,3 +7,11 @@ spack load @PIC_SPACK_COMPILER@ >/dev/null 2>&1 || echo "using default compiler"
 
 # required: PIConGPU and its dependencies
 spack load -r @PIC_SPACK_SPEC@
+
+# activate bash completion if available
+BASH_COMP_FILE=$PICSRC/bin/picongpu-completion.bash
+if [ -f $BASH_COMP_FILE ] ; then
+    source $BASH_COMP_FILE
+else
+    echo "bash completion file '$BASH_COMP_FILE' not found." >&2
```

Comment:
not sure if we should warn on this currently, only installs `@dev` will provide it so far and throw a confusing warning to all other (regular) users.

Reply:
If we do not show this warning it means we should never advertise this feature.
If a user has issues and wonders why he can not use the auto-completion he maybe learned about in a workshop or the docu it is very hard to find the reason.

Reply:
@ax3l could it be that you like to say we should skip this warning because in spack we currently only ship the master branch of picongpu but bash-completion files were only available in the dev branch?

Reply:
What I mean is: in Spack we tack some versions that are older and cannot have this file anyway and the `dev` branch that will always have this file.
Since the user can change little about this, I would not warn here. Basically newer releases have auto-completion, older releases don't.